Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -754,21 +754,34 @@ CGPathRef CGPathCreateCopyByStrokingPath(
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Caveat
+ @Notes Creates a mutable copy
 */
 CGPathRef CGPathCreateCopyByTransformingPath(CGPathRef path, const CGAffineTransform* transform) {
-    UNIMPLEMENTED();
-    return StubReturn();
+    return CGPathCreateMutableCopyByTransformingPath(path, transform);
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Interoperable
 */
 CGMutablePathRef CGPathCreateMutableCopyByTransformingPath(CGPathRef path, const CGAffineTransform* transform) {
-    UNIMPLEMENTED();
-    return StubReturn();
+    RETURN_NULL_IF(!path);
+
+    if (transform) {
+        if (CGAffineTransformEqualToTransform(*transform, CGAffineTransformIdentity)) {
+            return CGPathCreateMutableCopy(path);
+        }
+
+        CGMutablePathRef transformedPath = CGPathCreateMutable();
+        path->ClosePath();
+
+        FAIL_FAST_IF_FAILED(transformedPath->AddGeometryToPathWithTransformation(path->GetPathGeometry().Get(), transform));
```

Comment:
It bothers me a bit that GetPathGeometry returns a `ComPtr<ID2D1Geometry>`; it's an unnecessary +1/-1/+1 for this case (and an unnecessary +1/-1 for every other case where you call it (!)